gengokit/template/NAME-service/NAME-server/server_main.gotemplate

```diff
@@ -74,7 +74,7 @@ func main() {
 	// Wrap Endpoints with middlewares. See ../middlewares/endpoints.go
 	endpoints := svc.Endpoints{
 	{{range $i := .Service.Methods -}}
-		{{$i.Name}}Endpoint:    middlewares.WrapEndpoint({{ToLower $i.Name}}Endpoint),
+		{{$i.Name}}Endpoint:    {{ToLower $i.Name}}Endpoint,
```

Comment:
[This indentation will be wrong](https://play.golang.org/p/AFy7WE42_6), as output from the template execution. Template macros will not be rendered, so do not indent the inner portions of things like `{{ range }}` loops. I kind of want to say that the generated code gets gofmt'd, in which case this is fine since it improves readability of the template, and the rendered code will still be formatted properly, but I could't find evidence of the output code being gofmt'd. Maybe I didn't know what to look for.

If it's not going to be gofmt'd, we should, as part of the automated testing, run `gofmt -l` on the generated files, and filenames that are returned did not render in gofmt-standard style. Just making sure the generated files get gofmt'd would be just as good, or better, and would allow for the clarifying indentations in the template definitions.